### DISCUSSION TITLE
Logging SOAP requests/responses using WebServiceTemplate

### DISCUSSION THREAD

**Discussion by mslachtova** [zalando/logbook#1824] [Q&A]
Hi

I'm looking for a way to log external SOAP services using the `WebServiceTemplate`. Is there something similar to `LogbookClientHttpRequestInterceptor` for `RestTemplate`?

Thanks in advance

**Comment by kasmarian** [comment]
This depends on what are you using for sending these SOAP requests. Potentially you can use httpclient5 with logbook-httpclient5 and this will work as you'll add the interceptor to the http client directly.
